pages/code-review.md

```diff
@@ -1,6 +1,143 @@
 ---
 permalink: /code-review/
-title: Code Review
+title: Code Review Guide
```

Comment:
It would be great if the URLS of the pages matched their titles - perhaps change permalink and file name to match this?